woocommerce/google-listings-and-ads
# [GTIN] Migration notice

### Changes proposed in this Pull Request:



Closes part of https://github.com/woocommerce/google-listings-and-ads/issues/2616 as part of pcTzPl-2p2-p2.

Related to https://github.com/woocommerce/google-listings-and-ads/pull/2655

### Screenshots:


https://github.com/user-attachments/assets/fbff5990-6b4a-4e57-994b-89a3269c644c

<img width="812" alt="Screenshot 2024-10-29 at 20 48 00" src="https://github.com/user-attachments/assets/ee016764-5605-4295-87eb-ee6b42713366">




### Detailed test instructions:


1. Set this in console to view the events `localStorage.setItem( 'debug', 'wc-admin:*' )`
2. Enter into Google for WooCommerce (any page)
3. See the banner 
4. Click on "click here." and the event `wcadmin_gla_gtin_migration_banner_click` 
5. See the modal opening and event  `wcadmin_gla_modal_open`
6. Close the modal and see event `wcadmin_gla_modal_closed`
7. Repeat step 4
8. Click on "Start Migration" in the modal and event `wcadmin_gla_gtin_migration_banner_migration_start`
9. See the message "GTIN Migration was successfully scheduled" and event `wcadmin_gla_gtin_migration_banner_migration_scheduled`
10. Verify the job `gla/jobs/migrate_gtin/create_batch` is scheduled. Dont run it
11. Delete the option 
12. Repeat steps 7-8
13. See the message "Unable to start GTIN Migration" and event `wcadmin_gla_gtin_migration_banner_migration_failed`


### Additional details:


### Changelog entry

> Add - Banner for GTIN MIgration
